### PR TITLE
AdminLayout2: Fix sidepanel

### PR DESCRIPTION
**Pull request by daltonfury42**
Check preview, cleaned up sidepanel.

### CODE REVIEW

File: simplq/src/components/common/SidePanel/SidePanelItem.jsx [hunk 1]
```diff
@@ -33,17 +27,30 @@ const SidePanelItem = ({
...
+    if (expandable) {
+      setOpen(!open);
+    }
+    if (onClick) {
+      onClick();
+    }
+  };
 
   return (
-    <button type="button" className={styles['side-panel-item']} onClick={onClick}>
```
Comment (daltonfury42) [inline simplq/src/components/common/SidePanel/SidePanelItem.jsx:39]: remove button, as it a nasty react warning when there is a button inside a button, which we have.

File: simplq/src/components/common/SidePanel/SidePanelItem.jsx [hunk 2]
```diff
@@ -56,7 +63,7 @@ const SidePanelItem = ({
           )}
         </Collapse>
       )}
-    </button>
+    </div>
```
Comment (daltonfury42) [inline simplq/src/components/common/SidePanel/SidePanelItem.jsx:66]: Split the sidePanelItem into two, rough idea:

```
<SidePanelItem>
   <Header onClickToggle hoverEffects />
   < Collapse />
</SidePanelItem>
```

File: simplq/src/components/common/SidePanel/SidePanelItem.jsx [hunk 3]
```diff
@@ -4,14 +4,8 @@ import { Collapse } from '@material-ui/core';
 import PropagateLoader from 'react-spinners/PropagateLoader';
 import styles from '../../../styles/sidePanel.module.scss';
```
Comment (maaverik) [inline simplq/src/components/common/SidePanel/SidePanelItem.jsx:5]: Replace PropogateLoader with LoadingIndicator. It's the same thing.

Comment (daltonfury42) [reply simplq/src/components/common/SidePanel/SidePanelItem.jsx:4]: Done

**Review by maaverik — APPROVED** [review]

**Pull request merged**